osu.Game/Screens/Play/Player.cs

```diff
@@ -333,5 +344,22 @@ private void dimChanged(object sender, EventArgs e)
         private Bindable<bool> mouseWheelDisabled;
 
         protected override bool OnWheel(InputState state) => mouseWheelDisabled.Value && !IsPaused;
+
+        protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)

```

Comment:
Why is this not handled in hudOverlay? There should be basically no changes to Player.